client/yarn.lock

```diff
@@ -2054,6 +2054,11 @@
   "resolved" "https://registry.npmjs.org/abab/-/abab-2.0.5.tgz"
```

Comment:
Don't mix package managers; I'd recommend just sticking with `npm` - `yarn` has some advantages a while ago but `npm` has since caught up and is the preferred option